Configuration/DataProcessing/python/Impl/ppEra_Run2_2017_ppRef.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+"""
+_ppEra_Run2_2017_ppRef_
+
+Scenario supporting proton collisions
+
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Reco import Reco
+import FWCore.ParameterSet.Config as cms
+from Configuration.Eras.Era_Run2_2017_ppRef_cff import Run2_2017_ppRef
+
+from Configuration.DataProcessing.Impl.pp import pp
+
+class ppEra_Run2_2017_ppRef(pp):
+    def __init__(self):
+        pp.__init__(self)
+        self.recoSeq=''
+        self.cbSc='pp'
+        self.addEI=True
+        self.isRepacked=True
+        self.eras=Run2_2017_ppRef
+        self.promptCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2017_ppRef' ]
+        self.expressCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2017_ppRef' ]
+        self.visCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2017_ppRef' ]
```

Comment:
let's reuse the available customisePostEra_Run2_2017

Reply:
nm, I actually opted to make a new method for Run2_2017_pp_on_XeXe.
so, no change